### DISCUSSION TITLE
Bringing back previous behavior of CTRL+SHIFT+L

### DISCUSSION THREAD

**Discussion by benzino77** [zen-browser/desktop#4018] [Q&A]
Hi there,

With the https://github.com/zen-browser/desktop/commit/36dad7575063afd1ff672a8462f5680e1030d83c commit the CTRL+SHIFT+L shortcut now opens Dev Tools. The previous behavior autofill login/password fields by Bitwarden extension and automatically copy TOTP code. 

The question is: Is it possible to bring back the previous behavior of the shortcut?

**Comment by dcplaut** [comment]
I have exactly the same issue.  Is it at least possible to include this binding among those that can be modified in Settings > Keyboard shortcuts?  Or is there some addition/modification of json code that will disable the current binding?

**Reply by axel37** [reply]
Using bitwarden like the author, it's incredibly frustrating that this shortcut was changed without any indications on how to change it back !

It's not in the "keyboards shortcut" settings, and so far I haven't been able to find how to do it....

**Reply by Midda-C** [reply]
Yes, I've got years of muscle-memory hitting Ctrl+Shift+L to autofill with Bitwarden, and it's suddenly no longer working and is going into the debug tools.

Would really appreciate this being included in the Keyboard Shortcuts so I can disable/change it.

**Comment by dcplaut** [comment]
Looks like the most recent "twilight" version (1.6t) allows developer keybindings (including Ctrl-Shift-L) to be modified/disabled under Settings > Keyboard Shortcuts. 👍

**Comment by rnhurt** [comment]
These keyboard shortcut updates are very confusing.  It took me a minute to figure out that I wasn't crazy or that Bitwarden wasn't messed up.  Thank goodness you can update the DevTools shortcut now.
